Source/DafnyLanguageServer/DafnyRequestNames.cs

```diff
@@ -3,5 +3,6 @@ public static class DafnyRequestNames {
     public const string CounterExample = "dafny/counterExample";
     public const string CompilationStatus = "dafny/compilation/status";
     public const string GhostDiagnostics = "dafny/ghost/diagnostics";
+    public const string VerificationDiagnostics = "dafny/verification/diagnostics";
```

Comment:
This is Public API so, it should be our first and only change to decide what it looks like. Please comment on that.